### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by oliverlloyd**
Wrote a more useful readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,47 @@
-## Atoms Rendering
+# Atoms Rendering
+
+## What is an Atom?
+
+An Atom is a self contained piece of content that can be inserted into multiple articles. There are several different types of atoms, this repositry aims to document them all.
```
Comment (webb04) [inline README.md:5]: nitpick... `repository `

File: README.md [hunk 2]
```diff
@@ -1 +1,47 @@
...
+4. Publishing a new version of the library to Npm (see below)
+
+## Publishing
+
+Manual publishing steps:
+
+1. Ensure your changes are on master
+2. `yarn build`
+3. Create a branch, `yourname/v1.0.1`
+4. `yarn publish` (enter new version number, eg. 1.0.1)
```
Comment (webb04) [inline README.md:45]: Would it be useful to have a note about the npm credentials required?

File: README.md [hunk 3]
```diff
@@ -1 +1,47 @@
...
+```
+import { TheAtomYouWant } from '@guardian/atoms-rendering';
+
+<TheAtomYouWant someProp={localData.someProp} />
+```
+
+## Running locally
+
+```
+git clone https://github.com/guardian/atoms-rendering.git
```
Comment (JamieB-gu) [inline README.md:24]: SSH cloning also available.

**Comment by shtukas** [comment]
+1 ^^

**Pull request merged**